bolt/lib/Rewrite/BinaryPassManager.cpp

```diff
@@ -154,6 +155,11 @@ static cl::opt<bool>
                       cl::desc("print profile quality/bias analysis"),
                       cl::cat(BoltCategory));
 
+static cl::opt<bool> PrintContinuityStats(
+    "print-continuity-stats",
+    cl::desc("print profile function CFG continuity stats"),
+    cl::cat(BoltCategory));
```

Comment:
If the pass has low overhead and prints a single line by default, I suggest we enable it by default and drop the option. Make sure it doesn't print anything when there is no profile available.